TKGame/TKGame.cs

```diff
@@ -1,30 +1,40 @@
 ﻿using Microsoft.Xna.Framework;
 using Microsoft.Xna.Framework.Graphics;
 using Microsoft.Xna.Framework.Input;
+using Microsoft.Xna.Framework.Media;
+using System;
 using System.Collections.Generic;
 
 namespace TKGame
 {
     public class TKGame : Game
     {
+        public static TKGame Instance { get; private set; }
+        public static Viewport Viewport { get { return Instance.GraphicsDevice.Viewport; } }
+        public static Vector2 ScreenSize { get { return new Vector2(Viewport.Width, Viewport.Height); } }
+        public static GameTime GameTime { get; private set; }
+
         private GraphicsDeviceManager graphics;
         private SpriteBatch spriteBatch;
 
         // TODO: Refactor out of the main TKGame class
         List<Wall> walls;
         int screenWidth, screenHeight;
+        bool paused = false;
 
         public TKGame()
         {
+            Instance = this;
             graphics = new GraphicsDeviceManager(this);
             graphics.PreferredBackBufferWidth = 1600;
             graphics.PreferredBackBufferHeight = 900;
-            Content.RootDirectory = "Content";
             IsMouseVisible = true;
         }
 
         protected override void Initialize()
         {
+            this.Content.RootDirectory = "Content";
```

Comment:
This should go back to the constructor for `TKGame`. It's there by default with the template and I don't see any reason to change it

Reply:
If the documentation says to have it in there then we can move it. The example the documentation links to has it in the initialize function so I moved it over.

Reply:
Interesting. It truthfully doesn't matter that much, but makes sense to just keep it where it was imo. Also I was going based off this: https://docs.monogame.net/articles/getting_started/3_understanding_the_code.html